### ISSUE TITLE
Keeps crashing since last android update

### ISSUE THREAD

**Issue by hagelschaden** [SubhamTyagi/Last-Launcher#140]
I love this launcher but I cant use it anymore, because it keeps crashing all the time. Mostly when I close another app then I am back to my OnePlus launcher. The most annoying part is, that it also resets the standard launcher to none.

I am using the Oneplus 6 and this has startet since the last upgrade. Any ideas how to fix this?

**Comment by hagelschaden** [comment]
I'm assuming my phone has to be rooted for me to be able to see that folder. But sadly it is not :(

**Comment by SubhamTyagi** [comment]
> I'm assuming my phone has to be rooted for me to be able to see that folder. But sadly it is not :(

No, Root is not required.  Just go to external storage you will see the 'android' folder.

**Comment by hagelschaden** [comment]
Strange. There is an Amdroid folder, but data is completely empty. There also exists a hidden .android folder. Here data has lots of folders but none is called like the one you mentioned. None is even called last laucher.

**Comment by SubhamTyagi** [comment]
> Strange. There is an Amdroid folder, but data is completely empty. There also exists a hidden .android folder. Here data has lots of folders but none is called like the one you mentioned. None is even called last laucher.

If launcher(debug apk) crashes it will create that directory.

**Comment by SubhamTyagi** [comment]
I will try to do a convenient method for reporting crashes.

**Comment by hagelschaden** [comment]
Thanks! It keeps crashing but the folder is not being created.

Anyways I was thinking about installing Lineage OS today or tomorrow. Maybe this will solve the problem and Oxygen os is at fault here..

**Comment by SubhamTyagi** [comment]
Are you using debug apk file which is provided above?

**Comment by hagelschaden** [comment]
Of course! I made it the default launcher and it keeps displaying the same behavior but won't create the directory. Android/data is entirely empty there is not a single folder in there. Should I uninstall the other version?

**Comment by SubhamTyagi** [comment]
>Should I uninstall the other version?

Not necessary.
>Android/data is entirely empty there is not a single folder in there. 

Very strange, May be there hidden folder or in .android folders

**Comment by hagelschaden** [comment]
So the issue seems to have vanished with todays Android update. But thanks for trying to help. It would have been sad to uninstall this launcher. It's fantastic!
